modules/sdk-core/src/bitgo/staking/stakingWallet.ts

```diff
@@ -35,7 +35,9 @@ export class StakingWallet implements IStakingWallet {
   }
 
   get coin(): string {
-    return this.wallet.coin();
+    return (this.wallet.baseCoin as any).hasOwnProperty('tokenConfig')
```

Comment:
What are all the possibilities of `any` here. Can we define this?

Reply:
it's `ERC20Token` defined in sdk-coin-eth, but if I use that we would have a circular dependencies between sdk-core and sdk-coin-eth.

do you recommend to have a type define in sdk-core with just the `tokenConfig` in the baseCoin?

Reply:
I would suggest adding `tokenConfig?: BaseTokenConfig` to `IBaseCoin` instead.

Reply:
done, didn't know we had this in the sdk-core 👍